resources/cosmos-upload

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+echo "Enter CRN (default 1):"
+read -r input_crn
+crn=${input_crn:-1}
+
+echo "Enter SBI (default 1):"
+read -r input_sbi
+sbi=${input_sbi:-1}
+
+echo "Enter Database Name (default 'ffc-sfd-customer-receiver-messages'):"
+read -r input_database
+database=${input_database:-'ffc-sfd-customer-receiver-messages'}
+
+echo "Enter Container Name (default 'messages-container'):"
+read -r input_container
+container=${input_container:-'messages-container'}
+
+echo "Enter Quantity (default 1):"
+read -r input_quantity
+quantity=${input_quantity:-1}
```

Comment:
Question for my own understanding, does `quantity` refer to the number of items you'd want in the `messages-container` i.e. if the default is 1 it would just generate 1 item?

Reply:
Yep thats correct, do you think thats clear or do you have nay ideas for a better name? (Naming is hard)

Reply:
Hmm, nah I think it's best to stick with `Quantity`. 

As a follow up question, since the default for the `crn` and `sbi` are set to 1, if you choose to go with the default when running the `cosmos-upload` script, then the data will only be visible from the emulator/Postman right? It won't be visible from the frontend since we couldn't log into Defra Identity with a CRN of 1/choose a business with an SBI of 1?

Reply:
Yes thats right, the default for those is a bit pointless it was left in from when i was testing it out. It should probably be a requirement to input them.

Although if just testing to see if cosmos is working without using any frontend stuff it may not matter...

What do you think? I will ponder over night.

Reply:
The way it's set up now is solid. I don't see it as a requirement to make the default CRN and SBI one of the test ones that Defra ID have set up. More just asked to make sure I was thinking about it the right way. If I wanted to use the upload script to test the frontend then I would just choose one from the set of test data and input them anyway.